ownCloudAppShared/Client/File Lists/QueryFileListTableViewController.swift

```diff
@@ -480,6 +496,14 @@ open class QueryFileListTableViewController: FileListTableViewController, SortBa
 			}
 		}
 	}
+
+	override open func tableView(_ tableView: UITableView, shouldHighlightRowAt indexPath: IndexPath) -> Bool {
+		if let item = itemAt(indexPath: indexPath), item.type != .collection, isFolderSelectionOnlyMode {
```

Comment:
@hosy why is highlighting handled differently here?

Reply:
removed this code